ansible_collections/arista/cvp/plugins/modules/cv_device_v3.py

```diff
@@ -186,6 +186,9 @@ def main():
 
     if ansible_module.params['state'] == 'absent':
         ansible_module.fail_json(msg='State==absent is not yet supported !')
+    # Check if the device list is empty in case the device_filter is incorrectly set
+    if len(ansible_module.params['devices']) == 0:
+        ansible_module.fail_json(msg='Device list is empty. Please check if the device_filter is correct!')
```

Comment:
An error message in `arista.cvp` should not reference some variable in `arista.avd.eos_designs`. It could have been called with an empty list in many other ways.
I suggest changing the error message to `"Aborting, 'devices' list is empty"`